### PR TITLE
Add more fathom events

### PR DESCRIPTION
**Pull request by NakedFool**
## What does this PR do and why?

Closes https://github.com/jbx-protocol/juice-interface/issues/3005

I've created `fathomEvents` for storing all eventId constants that we now use, I've decided to categorize constants per page, for example, we have `TOP_NAV` which contains all CTAs that are only in the top nav of the application, that way in the future if we decide to add more events we can easily see where to add and it is more readable, lmk what you think of this approach!

One more thing, I think we need to remove the old event `Click Design Your Project CTA` since it used to be an event for tracking creating a project on the hero homepage, which we now replaced with a new eventId, lmk if I'm being correct here! 🍻 

EDIT: 
Hmm, I think something happened in the meantime, I was initially creating this branch from `main` but I'm seeing a lot of conflicts right now, let me see how to handle it.

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by tomquirk** [comment]
@NakedFool heads up, going to push a rebase!

File: src/components/Navbar/navigationItems.tsx [hunk 1]
```diff
@@ -130,7 +130,10 @@ export const desktopMenuItems = ({
     key: 'discord',
     label: (
       <Link href="https://discord.gg/wFTh4QnDzk">
-        <a className="flex cursor-pointer items-center font-medium text-black hover:opacity-70 dark:text-slate-100">{t`Discord`}</a>
+        <a
+          className="flex cursor-pointer items-center font-medium text-black hover:opacity-70 dark:text-slate-100"
+          onClick={() => trackFathomGoal('PKIBZYTS')}
```
Comment (tomquirk) [inline src/components/Navbar/navigationItems.tsx:135]: **suggestion** could we use the goal constant variable from the constants file here and throughout?

Comment (NakedFool) [reply src/components/Navbar/navigationItems.tsx]: I think we all good now! 🍻

**Review by tomquirk — APPROVED** [review]
Love your work @NakedFool , approving!

**Pull request merged**